DBbot/classes/column.ts

```diff
@@ -58,4 +58,87 @@ export class Column {
     public addOperator(operator: Operator): void {
         this.operatorsArray.push(operator);
     }
+
+    public mean(): number {
+        if (this.dataType === "string") {
+            throw new Error("Cannot calculate mean for string data type");
```

Comment:
I saw all the errors are hard-coded; in ideal words its shoud be in a resources file; I think here it is fine, but I just notice it

Reply:
fixed